src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskProvidingConfiguration.swift

```diff
@@ -66,10 +71,10 @@ extension RiskProvidingConfiguration {
 		return result
 	}
 
-	func manualExposureDetectionState(lastExposureDetectionDate detectionDate: Date?) -> ManualExposureDetectionState? {
+	func manualExposureDetectionState(activeTracingHours: Int, lastExposureDetectionDate detectionDate: Date?) -> ManualExposureDetectionState? {
```

Comment:
same comment as above with the `activeTracingHours` param

Reply:
:white_check_mark: